app/uk/gov/hmrc/agentsexternalstubsfrontend/views/main_template.scala.html

```diff
@@ -14,32 +14,66 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.play.views.html.layouts
-@import uk.gov.hmrc.agentsexternalstubsfrontend.views.html.govuk_wrapper
-@import play.api.Configuration
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukLayout, GovukBackLink}
+@import uk.gov.hmrc.govukfrontend.views.html.components.{BackLink, Text}
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.{HmrcBanner, Banner}
+@import uk.gov.hmrc.agentsexternalstubsfrontend.config.FrontendConfig
+@import uk.gov.hmrc.agentsexternalstubsfrontend.views.html.components.{header, head}
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.{HmrcStandardFooter, HmrcScripts, HmrcReportTechnicalIssueHelper}
+@import uk.gov.hmrc.govukfrontend.views.html.components.TwoThirdsMainContent
+@import uk.gov.hmrc.govukfrontend.views.html.components.TwoThirdsOneThirdMainContent
 
-@this(govUkWrapper: govuk_wrapper, uiSidebar: uk.gov.hmrc.play.views.html.layouts.Sidebar, article: layouts.Article)
+@this(
+        head: head,
+        header: header,
+        govukLayout: GovukLayout,
+        govukBackLink: GovukBackLink,
+        hmrcStandardFooter: HmrcStandardFooter,
+        hmrcScripts: HmrcScripts,
+        hmrcBanner : HmrcBanner,
+        hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+        twoThirdsLayout: TwoThirdsMainContent,
+        twoThirdsOneThirdLayout: TwoThirdsOneThirdMainContent
+)
 
 @(title: String,
-  sidebarLinks: Option[Html] = None,
-  contentHeader: Option[Html] = None,
-  bodyClasses: Option[String] = None,
-  mainClass: Option[String] = None,
-  scriptElem: Option[Html] = None)(mainContent: Html)(implicit messages: Messages, request : Request[_], configuration: Configuration)
+        backLinkHref: Option[String] = None,
+        wide: Boolean = false,
+        sidebar: Option[Html] = None // overrides the 'wide' option if present
+)(mainContent: Html)(implicit request : Request[_], messages: Messages)
 
-@serviceInfoContent = {}
+@beforeContent = {
+@hmrcBanner(Banner())
+@if(backLinkHref.nonEmpty){
+    <nav aria-label="Back Link">
+    @backLinkHref.map(url =>
+        govukBackLink(BackLink(href = url, content = Text(messages("button.back"))))
+    )
+    </nav>
+}
+}
+
+@mainContentWithHelpLink = {
```

Comment:
we don't need the Deskpro link in the stubs? I know it's there atm but I thought we'd just remove it here and feed mainContent at the bottom of govukLayout